src/components/gearGrid.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {Modal, Button} from 'react-bootstrap';
+import Selector, {WeaponCard} from './shared/card';
+import './builder.scss'
+import Stats from './stats.js'
+
+class GearSelection extends React.Component {
+    state={
+        itemClicked: "",
+        primary: {
+            name: "Baker's Dozen",
+            type: "rifle",
+            damage: 235060,
+            talent: {
+                name: "Perfect Lucky Shot", 
+                desc: "Magazine capacity is increased by 30%. Missed shots from cover have a 100% chance to return to the magazine.", 
+                icon: "luckyshot"
+            },
+            isNamed: true
+        },
+        secondary: {
+            name: "",
+            type: "",
+            damage: 0,
+            talent: {
+                name: "", 
+                desc: "", 
+                icon: ""
+            },
+            isNamed: false
+        },
+    }
+
+    gearGrid(props) {
+        const [show, setShow] = React.useState(false);
+
+        const handleClose = () => setShow(false);
+        const handleShow = () => setShow(true);
+
+        return (
+            <>
+                {props.state.itemClicked !== "" && (
+                    <div></div>
+                )}
+                <div className="createBuildGrid col-xs-12 col-lg-9 row">
+                    <Selector
+                        category="signature"
+                        isSelected
+                        specialization="sharpshooter"
+                    />
+                    <Selector
+                        category="gernade"
```

Comment:
```suggestion
                        category="grenade"
```
I wonder who wrote this part

Reply:
Will fix in the next commit